src/net/gateway-archive.ts

```diff
@@ -0,0 +1,151 @@
+import { FileApi } from "../api/file";
+import { GatewayArchiveError } from "../errors/gateway-archive";
+import { TextRecordKeys } from "../models/entity"
+import { VochainProcessStatus } from "../models/protobuf";
+import { getEnsPublicResolverByNetwork } from "../util/ens";
+import { ContentUri } from "../wrappers/content-uri"
+import { EthNetworkID, IGatewayClient, IGatewayDVoteClient } from "../common";
+
+export interface IArchiveResponseBody {
+    process?: {
+        [k: string]: any
+    }
+    results?: {
+        [k: string]: any
```

Comment:
Same, no type definition already for this?

Reply:
No, I haven't found a type definition for this. Should I create one?

Reply:
Aren't we using any types on `getDigestedResults` and `getRawResults` ?